lib/board.rb

```diff
@@ -1,113 +1,101 @@
 class Board
-  attr_accessor :grid 
 
   def initialize(symbols = Array.new(9))
     @grid = symbols
   end
 
   def is_empty
-    !(grid.include?(:X) or grid.include?(:O))
+    !(grid.include?(PlayerSymbols::X) or grid.include?(PlayerSymbols::O))
```

Comment:
There are a lot of convenience methods in ruby to simplify boolean logic. Maybe a more idiomatic solution would be: `grid.all?(&:nil?)`.

Also, the boolean operators `&&` and `||` are preferred over the keywords `and` and `or`. They have a stronger precedence and behave more regularly. From the ruby style guide:
  > The and and or keywords are banned. It's just not worth it. Always use && and || instead.